app/src/androidTest/java/ch/epfl/sweng/erpa/MainTests.java

```diff
@@ -0,0 +1,42 @@
+package ch.epfl.sweng.erpa;
```

Comment:
Can you please rename this file to be more expressive on what it does? Something like `MainActivityTest.java`